api/v1alpha3/vspheremachinetemplate_conversion.go

```diff
@@ -17,7 +17,10 @@ limitations under the License.
 package v1alpha3
 
 import (
+	apiconversion "k8s.io/apimachinery/pkg/conversion"
 	infrav1alpha4 "sigs.k8s.io/cluster-api-provider-vsphere/api/v1alpha4"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
+	clusterv1a4 "sigs.k8s.io/cluster-api/api/v1alpha4"
```

Comment:
```suggestion
	clusterv1a3 "sigs.k8s.io/cluster-api/api/v1alpha3"
	clusterv1a4 "sigs.k8s.io/cluster-api/api/v1alpha4"
```
nit: This is one of the few places we are using both the imports, makes sense to keep it clear.